tests/lib/nested.sh

```diff
@@ -1503,3 +1503,32 @@ nested_wait_for_device_initialized_change() {
         sleep "$wait"
     done
 }
+
+nested_build_ubuntu_image() {
```

Comment:
why is it a nested function? I see it being used for regular core testing

Reply:
moved it to separate image.sh file